docs/source/examples/interactive-development.rst

```diff
@@ -0,0 +1,170 @@
+Start a Development Cluster
+===========================
+
+SkyPilot makes interactive development easy on Kubernetes or cloud VMs. It helps you:
+
+#. Launch: Quickly get a cluster with GPU or other resource requirement with a single command.
+#. Autostop: Automatically stop the cluster after some idle time for cost savings.
+#. Connect: Easily connect to the cluster using the cluster name (e.g., ``ssh dev``).
+
+Launch
+------
+
+To launch a cluster with a cheap GPU for development:
+
+.. code-block:: bash
+
+  # Launch a cluster with 1 NVIDIA GPU and sync the local working directory to the
+  # cluster. This can be launched as a pod in your Kubernetes cluster or a cloud VM.
+  sky launch -c dev --gpus T4 --workdir .
```

Comment:
Idea: to make "This can be launched as a pod in your Kubernetes cluster or a cloud VM." more clear, we can take it out of comments, and put it as a normal text, followed by tabbed screenshots: K8s -> a screenshot of a pod named `sky .. dev` in K8s outputs/dashboard; cloud -> a screenshot of cloud console, etc.

Wdyt?

Reply:
Added. Thanks!